### ISSUE TITLE
check_ping on host without ipv6 will always return a warning 

### ISSUE THREAD

**Issue by DagNygren** [nagios-plugins/nagios-plugins#579]
The check_ping plugin will not pass the "-4" flag to /usr/sbin/ping when requesting ipv4 only. This will generate a text at stderr on a machine with no ipv6 support and that will be considered a warning.

**Comment by Vaevictusnet** [comment]
ping rolled ping6 into ping now, so this is actually a bug that affects the plugin.  shouldn't be too hard to resolve.

```
oskarblues /etc # /usr/lib/nagios/plugins/check_ping -H 192.0.0.3 -w 10,5% -c 20,10% -p 1 -4  -vvv
CMD: /bin/ping -n -U -w 10 -c 1 192.0.0.3
Output: PING 192.0.0.3 (192.0.0.3) 56(84) bytes of data.
Output: 64 bytes from 192.0.0.3: icmp_seq=1 ttl=254 time=0.275 ms
Output:
Output: --- 192.0.0.3 ping statistics ---
Output: 1 packets transmitted, 1 received, 0% packet loss, time 0ms
Output: rtt min/avg/max/mdev = 0.275/0.275/0.275/0.000 ms
Got stderr: /bin/ping: socket: Address family not supported by protocol
PING WARNING - System call sent warnings to stderr Packet loss = 0%, RTA = 0.28 ms|rta=0.275000ms;10.000000;20.000000;0.000000 pl=0%;5;10;0
10.000000:5% 20.000000:10%

[20210427-09:16:26]
oskarblues /etc # /usr/lib/nagios/plugins/check_ping -H 192.0.0.3 -w 10,5% -c 20,10% -p 1  -vvv
CMD: /bin/ping -n -U -w 10 -c 1 192.0.0.3
Output: PING 192.0.0.3 (192.0.0.3) 56(84) bytes of data.
Output: 64 bytes from 192.0.0.3: icmp_seq=1 ttl=254 time=0.302 ms
Output:
Output: --- 192.0.0.3 ping statistics ---
Output: 1 packets transmitted, 1 received, 0% packet loss, time 0ms
Output: rtt min/avg/max/mdev = 0.302/0.302/0.302/0.000 ms
Got stderr: /bin/ping: socket: Address family not supported by protocol
PING WARNING - System call sent warnings to stderr Packet loss = 0%, RTA = 0.30 ms|rta=0.302000ms;10.000000;20.000000;0.000000 pl=0%;5;10;0
10.000000:5% 20.000000:10%

[20210427-09:16:30]



```

**Comment by handsfoldedpupil** [comment]
I came to this page because I ran into the problem with check_ping error "PING WARNING - System call sent warnings to stderr Packet loss".

The solution seems to be for people with ipv4 only network by passing the option below to configure and rebuild nagios plugins:

./configure  --with-ping-command="/bin/ping -4 -n -U -w %d -c %d %s"
